src/pkg/count/keys.go

```diff
@@ -3,10 +3,12 @@ package count
 type key string
 
 const (
+	APICallTokensConsumed key = "api-call-tokens-consumed"
```

Comment:
Could you add a small description for each of these?

Reply:
Sure, why not.